docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
 ---
-title: Learn About Event Driven Architecture
+title: About Event Driven Architecture at VA
 ---
 
 # Introduction
-The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at the VA. Event-driven architecture uses events, which are specific instances of something that happened, to communicate with systems that are subscribed to the stream of events. The systems that are producing the events [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
+The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at VA. Event-driven architecture uses events – types of actions, such as a Veteran creating a medical appointment or updating their beneficiaries – to communicate with systems that are subscribed to the stream of events. The systems that are producing the events, also known as [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events, also known as [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
```

Comment:
In the newly added section above, replace the en dash (–) with an em dash (—) and remove the spaces on either sides of the dash. I'll add this to our style guide.